drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1054,11 +1062,13 @@ void RasterizerSceneGLES3::_filter_sky_radiance(Sky *p_sky, int p_base_layer) {
 
 	CubemapFilterShaderGLES3::ShaderVariant mode = CubemapFilterShaderGLES3::MODE_DEFAULT;
 
-	if (p_base_layer == 0) {
+#ifdef GL_API_ENABLED
+	if (RasterizerGLES3::is_gles_over_gl() && p_base_layer == 0) {
```

Comment:
I don't think this is correct. We need the mipmaps generated because we want to use the mipmapped version of the base layer for calculating the radiance map. This is the "pre-filtering" part of pre-filtered importance sampling

Reply:
Ah my bad, we do this check in `_update_dirty_skys` probably because `glTexImage2D` isn't supported under GLES. It is interesting here that for GLES `glGenerateMipmap` is also skipped (that might be a bug in those places as it won't allocate space for the mipmaps)

I made the same change here to test if it would fix the issue and fall back on the old approach of filtering, I didn't change it back.

Would be good to have a discussion on why this is working the way it does in `_update_dirty_skys`

Reply:
Definitely works fine on my Quest with this returned the way it was.